### PR TITLE
Add Profile Completion processor 

### PR DESCRIPTION
**Pull request by Schpotsky**
Challenge https://www.topcoder.com/challenges/7f186c09-77f9-421e-b99b-20463c65f4d0 updates

### CODE REVIEW

File: src/common/constants.js [hunk 1]
```diff
@@ -36,5 +41,13 @@ module.exports = {
     5: 'payoneer',
     6: 'western union',
     7: 'wipro payroll'
+  },
+
+  // The mapping between the profile completion metadata field name and the traits id
+  // The keys are the trait ids, the values are the corresponding field in the metadata object
+  TRAITS_TO_PROFILE_COMPLETION_CHECKLIST_METADATA_MAP: {
```
Comment (vikasrohit) [inline src/common/constants.js:48]: @Schpotsky does it need more mappings for skills, profile_picture, bio etc? or Am I reading it wrong?

Comment (vikasrohit) [reply src/common/constants.js:48]: I think I got it. The other fields that I mentioned are not determined by traits, rather they are either direct property of the member model or entirely different model (skills).

**Review by vikasrohit — APPROVED** [review]
LGTM

**Comment by vikasrohit** [comment]
@Schpotsky I am merging the PR, however, I would like to know your answer to my last question.

**Comment by vikasrohit** [comment]
Yep, totally in consensus with that. There was no other way to handle in that duration. Just wanted to keep it in my know hows of the onboarding processor. We can handle it later. Thanks for the update.

**Comment by vikasrohit** [comment]
THanks @Schpotsky for the answer. Just curious how did you and reviewers verify the profile_picture flag? Generating a mock event?

**Comment by vikasrohit** [comment]
@veshu can you please look at the [member microservice code](https://github.com/appirio-tech/ap-member-microservice/) to see why it is not generating `photoUrl` in the event payload when profile picture is uploaded by user? if there is something we can fix, we would fix it otherwise we can fetch the `photoUrl` of the user by making explicit API call to member api.

**Comment by Schpotsky** [comment]
> THanks @Schpotsky for the answer. Just curious how did you and reviewers verify the profile_picture flag? Generating a mock event?

Yes, we are sending mock events directly to Kafka topic

**Comment by vikasrohit** [comment]
@Schpotsky @veshu I guess we need to capture `member.action.profile.photo.create` and `member.action.profile.photo.update` event for profile picture update.

**Comment by vikasrohit** [comment]
I am not sure where `member.action.profile.photo.create` is used, I am trying to verify if that is thrown on a new user or not. Sample payload for `member.action.profile.photo.update` is 
```
{
  "userId": 22688955,
  "photoURL": "https://xxx.amazonaws.com/member/profile/abacd.png",
  "updatedAt": "2021-10-11T04:53:22.048Z",
  "updatedBy": "22688955"
}
```

**Comment by vikasrohit** [comment]
Copied discussion to https://github.com/topcoder-platform/onboarding-processor/issues/13#issuecomment-939684622

**Pull request merged**